server/server.js

```diff
@@ -1,12 +1,41 @@
 /* eslint no-console:0 */
 var ServiceManager = require('./service-manager');
+var assert = require('assert');
 var boot = require('loopback-boot');
 var debug = require('debug')('strong-mesh-models:server');
 var loopback = require('loopback');
+var MinkeLite = require('minkelite');
 
-function server(serviceManager) {
+function server(serviceManager, options) {
   var app = module.exports = loopback();
   app.serviceManager = serviceManager;
+  options = options || {};
+
+  for (var k in options) {
+    if (!options.hasOwnProperty(k)) continue;
+    app.set(k, options[k]);
+  }
+
+  assert(!app.get('trace.db.inMemory') && !app.get('trace.db.path'),
```

Comment:
Ya, I see it now.. I think it's the assert instead of `if (...)` that threw me off.